src/utils/fetchClient.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://mate.academy/students-api';
+
+export function wait(delay: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
+}
+
+type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
+
+function request<T>(
+  url: string,
+  method: RequestMethod = 'GET',
+  data: any = null, // we can send any data to the server
```

Comment:
Replace 'any' with a specific type or 'unknown' for the 'data' parameter.

Reply:
<img width="616" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/17066980/f143793e-67fa-4a08-93cf-2c0041163626">